app/models/complaint.rb

```diff
@@ -97,7 +97,9 @@ def ok_to_sum?
   end
 
   def set_facility_code_complaint
+    # rubocop:disable Performance/RegexpMatch
     COMPLAINT_COLUMNS.each_pair { |complaint, issue_regex| self[complaint] = issues =~ issue_regex ? 1 : 0 }
```

Comment:
Why the exemption?

https://www.rubydoc.info/github/rubocop-hq/rubocop/master/RuboCop/Cop/Performance/RegexpMatch

After reading ☝️ seems like you want to just use `issues.match?(issue_regex)` - or am I missing something?

Reply:
Plan is to circle back and fix all of the `=~` in future work

Reply:
Were there other `=~` errors caught? I dont see any in this PR.

Reply:
There are 6 usages of `=~` throughout the repo, not all of them were caught by rubocop

Reply:
Is rubocop is misconfigured? 😬 or is missing these other usages intentional? Maybe a separate bug is warranted.

I'd vote for just fixing in this case becayse "future work" == "never".

Reply:
changed